contracts/gateway-v1.yaml

```diff
@@ -2136,6 +2136,125 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+
+  /hip-links/link:
+    post:
+      tags:
+        - link
+        - hip facing
+      summary: Link patient's care contexts
+      description: >
+        Request from Hip to link a patient's care contexts
+          1. **Validate account reference number and care context reference number**
+          2. **Before eventual link confirmation, CM needs to authenticate the request with the patient**
```

Comment:
Does it require auth? Thought we decided that we will trust that the hospital is trusted. Otherwise for many users, who don't have mobile number (e.g. some govt services), the linking would never be done.

Reply:
When we call the CM or Hip bridge from gateway. I think there should be some sort of auth